### ISSUE TITLE
Item 10, 3rd Edition, from learning.oreilly.com: determine default encoding using incorrect function name...

### ISSUE THREAD

**Issue by KjellSwedin** [bslatkin/effectivepython#122]
The lesson here is that you should check the default encoding on your system (using python3 -c 'import locale; print(locale.getpreferred-encoding())') to understand how it differs from your expectations. 

should be:

The lesson here is that you should check the default encoding on your system (using python3 -c 'import locale; print(locale.getpreferredencoding())') to understand how it differs from your expectations. 

-------------------------------------
As written:
kjells:notebooks$ python3 -c 'import locale; print(locale.getpreferred-encoding())'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'locale' has no attribute 'getpreferred'

With change:
kjells:notebooks$ python3 -c 'import locale; print(locale.getpreferredencoding())'
UTF-8

kjells:notebooks$ python3 --version
Python 3.12.3

**Comment by bslatkin** [comment]
Yes this typo snuck in there! Thank you for the report. We'll get it fixed.
